### PR TITLE
Add missing keywords; alphabetize

### LINKED ISSUE #2132: dart language missing keywords
Some new keywords have been introduced recently:

* covariant deferred extension Function hide interface mixin on show

Some of these were just missed previously (like deferred). Some are recent (mixin), and one is upcoming in a release this year (extension).

### PR DESCRIPTION
**Pull request by srawlins**
Fixes #2132

### CODE REVIEW

**Comment by marcoscaceres** [comment]
Oops, looks  like something broke.

File: src/languages/dart.js [hunk 1]
```diff
@@ -64,15 +64,16 @@ function (hljs) {
   ];
 
   var KEYWORDS = {
-    keyword: 'assert async await break case catch class const continue default do else enum extends false final ' +
-      'finally for if in is new null rethrow return super switch sync this throw true try var void while with yield ' +
-      'abstract as dynamic export external factory get implements import library operator part set static typedef',
+    keyword: 'abstract as assert async await break case catch class const continue covariant default deferred do ' +
+      'dynamic else enum export extends extension external factory false final finally for Function get hide if ' +
+      'implements import in inferface is library mixin new null on operator part rethrow return set show static ' +
+    'super switch sync this throw true try typedef var void while with yield'
```
Comment (marcoscaceres) [inline src/languages/dart.js]: Whitespace and missing ,

Comment (srawlins) [reply src/languages/dart.js]: Eek, sorry. Fixed.

**Comment by marcoscaceres** [comment]
Thanks for the fixes @srawlins! Really appreciate it.

**Comment by marcoscaceres** [comment]
Will package up a release in the next few days.

**Pull request merged**